usaspending_api/api_contracts/contracts/v2/autocomplete/city.md

```diff
@@ -11,23 +11,31 @@ List of cities matching search criteria
 
 + Request (application/json)
     + Attributes (object)
-        + `search_text`: `Springfield` (required, string)
-        + `limit`: 40 (required, number)
-        + `filter` (object)
-            + `country_code`: `USA` (required, string)
-            + `scope`: `recipient_location` (required, enum[string])
+        + `search_text` (required, string)
+        + `limit` (required, number)
+        + `filter` (optional, object)
+            + `country_code` (required, string)
+            + `scope` (required, enum[string])
                 + Members
                     + `primary_place_of_performance`
                     + `recipient_location`
-            + `state_code`: `VA` (optional, string)
+            + `state_code` (optional, string)
+    + Body
+
+            {
+                "search_text": "Springfield",
+                "limit": 40,
+                "filter": {
+                    "country_code": "USA",
+                    "scope": "recipient_location",
+                    "state_code": "VA"
+                }
+            }
 
 + Response 200 (application/json)
     + Attributes (object)
-        + `count`: 10 (required, number)
-        + `results` (required, array[AutocompleteCityResult], fixed-type)
-
-# Data Structures
-
-## AutocompleteCityResult (object)
-+ `city_name`: `Springfield` (required, string)
-+ `state_code`: `VA` (required, string, nullable)
+        + `count`(required, number)
```

Comment:
Do you think it is cleaner to define the object "in-line" of the response vs defining a stand-alone object?

Reply:
Made some of these changes since it seemed wasteful to have an object defined below when it isn't being reused and is basically the whole response. If you think it should be undone happy to do so. Not leaning strong one way or the other.

Reply:
Previous discussions state that the entire response shouldn't be defined as an object, but if the response is an array of objects it is acceptable to define the object separately. As long as the response array can be defined as fixed-type there are no issues.

Leave the code as-is.